src/status_im/protocol/handlers.cljs

```diff
@@ -492,15 +492,13 @@
 
 (handlers/register-handler-fx
   :update-message-status
-  [re-frame/trim-v (re-frame/inject-cofx :get-stored-message)]
-  (fn [{:keys [db get-stored-message]} [{:keys [from sent-from payload]} status]]
+  [re-frame/trim-v]
+  (fn [{:keys [db]} [{:keys [from sent-from payload]} status]]
     (let [message-identifier (get-message-id payload)
           chat-identifier    (or (:group-id payload) from)
           message-db-path    [:chats chat-identifier :messages message-identifier]
           from-id            (or sent-from from)
-          message            (or (get-in db message-db-path)
-                                 (and (get (:not-loaded-message-ids db) message-identifier)
-                                      (get-stored-message message-identifier)))]
+          message            (get-in db message-db-path)]
```

Comment:
Similar point like before, we only have 20 messages loaded for each chat when it's opened (more when one scrolls up), so whenever we receive update for some message which is not yet loaded (second `and` condition in the original expression), we should update it as well, otherwise we ignore status updates for just little bit older messages.